package.json

```diff
@@ -14,6 +14,7 @@
     "lint:js": "eslint config src",
     "lint:js:fix": "eslint config src --fix",
     "lint:sass": "stylelint 'src/**/*.scss' --config .stylelintrc.json",
+    "patch:hosts": "fec patch-etc-hosts",
```

Comment:
Is it a requirement for the fix?

Reply:
@r14chandra no but this script will be helpful for developers as it is needed as initial setup before running code locally.